src/shared/specifier.c

```diff
@@ -97,7 +101,10 @@ int specifier_printf(const char *text, const Specifier table[], void *userdata,
 
                                         ret = n;
                                         t = n + j + k;
-                                } else {
+                                } else if (strchr(POSSIBLE_SPECIFIERS, *f))
+                                        /* Oops, an unknown specifier. */
+                                        return -EINVAL;
```

Comment:
maybe make this return some recognizable error? -ESRCH or so?

Reply:
ESRCH could also be returned if a user or group is not found. I changed it to EBADSLT.